next.config.js

```diff
@@ -1,5 +1,11 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  // //add images loaded by NextJS; currently a premature optimization
+  // images:{
+  //   loader: 'penn-state-logo',
+  //   loaderFile: ''
+
+  // },
```

Comment:
Remove these comments, and when implementing the content. Ensure that the images are being locally sourced from within a public folder. 

[https://nextjs.org/docs/app/building-your-application/optimizing/static-assets](url)